src/translate.js

```diff
@@ -869,11 +869,30 @@ var multiLangData = {
         "zh": "DMG Cap",
     },
     "マスボダメ上限説明": {
-        "en": "The value of Job Master bonus \"DMG Cap+○○%\"",
+        "en": "The value of Job Master bonus \"DMG Cap+**%\"",
         "ja": "ジョブマスターボーナスの\"ダメージ上限+○○％\"の値です\n(各ジョブごとのボーナスとは別です)",
         "zh": "职业满级Bonus中\"DMG Cap+○○％\"中的值\n(和各职业的Bonus是独立的)",
     },
-
+    "マスボ奥義ゲージ": {
+        "en": "Normal Attack Charge Bar",
```

Comment:
This English key might be confusing. Can you update this as "One-foe Attacks Bonus Charge" as it is how rephrased in the game?

Reply:
Thank you for the advice. My english skill is low, so I was saved.